tox.ini

```diff
@@ -142,9 +140,13 @@ basepython=python2.7
 [testenv:manifest-checker]
 skip_install = true
 
-# TwistedChecker needs to be 3.5
 [testenv:lint]
-basepython=python3.5
+skip_install=True
+basepython=python3.6
+
+[testenv:black]
+skip_install=True
+basepython=python3.6
 
```

Comment:
FWIW I found it useful to add a test rnv `black-reformat` to that it's easy for a developer to reformat code to comply with `black`.  Example [here](https://github.com/burningmantech/ranger-deploy/blob/master/tox.ini#L71).